src/c_commands.cpp

```diff
@@ -63,6 +65,11 @@ static void CmdSet(CommandArguments args)
     }
 }
 
+static void CmdQuit(CommandArguments args)
+{
+    exit(0);
```

Comment:
This really needs to be `I_Quit`.  That function will run necessary bookkeeping functions upon exiting.